url_checker/url_checker_lib.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Initialization
+ */
+function url_checker_init() {
+  // Only run status check on live deployments
+  if ($_ENV['PANTHEON_ENVIRONMENT'] != 'live') {
+    die();
```

Comment:
A `die` with no message can lead to confusion. Link checking should not happen on the live environment; usually, testing is done in the test environment prior to deploying. More generally, someone might want to test the script in the dev environment just to see how it works, and might (hypothetically :smile: ) not notice the 'live' environment check. Significant "why is this not working" debugging might follow.

I think that this is a good example of what @marktheunissen was concerned about vis-a-vis "hiding" example script functionality in include files. Even small files such as this one are prone to encapsulation problems, where the first-time experimenter becomes more likely to assume that the function is something that (a) just works (b) has no configuration and (c) has no side-effects.